d5005/scripts/setup-bsp.py

```diff
@@ -202,23 +204,22 @@ def setup_bsp(bsp_root, env_vars, bsp, verbose):
     
     #symlink the contents of bsp_dir/build into syn_top
     BSP_BUILD_DIR_FILES=os.path.join(bsp_qsf_dir, '*')
-    ASP_BUILD_DIR_SYMLINK_CMD="cd " + QUARTUS_SYN_DIR + " && ln -s " + ASP_BUILD_DIR_RELATIVE_TO_QUARTUS_BUILD_DIR + "/* ."
+    ASP_BUILD_DIR_SYMLINK_CMD="cd " + QUARTUS_SYN_DIR + " && ln -sf " + ASP_BUILD_DIR_RELATIVE_TO_QUARTUS_BUILD_DIR + "/* ."
     run_cmd(ASP_BUILD_DIR_SYMLINK_CMD)
     
-    #symlink the (i)ofs_top.qpf and (i)ofs_pr_afu.qsf file to bsp_dir
-    if "n6001" in bsp:
-        PR_AFU_QSF_FILENAME="ofs_pr_afu.qsf"
-        PR_AFU_QPF_FILENAME="ofs_top.qpf"
-    else:
-        PR_AFU_QSF_FILENAME="iofs_pr_afu.qsf"
-        PR_AFU_QPF_FILENAME="d5005.qpf"
-        
-    rm_glob(os.path.join(bsp_dir, PR_AFU_QSF_FILENAME))
-    OFS_PR_AFU_QSF_SYMLINK_CMD="cd " + bsp_dir + " && ln -s " + QUARTUS_BUILD_DIR_RELATIVE_TO_KERNEL_BUILD_DIR + "/" + PR_AFU_QSF_FILENAME + " ."
+    PR_AFU_QPF_FILENAME=os.path.basename(glob.glob(QUARTUS_SYN_DIR + "/*.qpf")[0])
```

Comment:
Find the qpf and *pr_afu.qsf files from the pr-build-template's syn_top/ folder, rather than hard-code them based on d5005/n6001.